src/tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -246,6 +245,12 @@ public override bool Execute ()
             config.Extra[name] = valueObject;
         }
 
+        if (!InvariantGlobalization)
+        {
+            string? icuDictionary = File.ReadAllText(IcuDictionary!);
```

Comment:
Since this read operation and the deserialize operation below can fail for a variety of possible reasons, it might be nice to wrap them in a try/catch that adds diagnostic info or guidance on failure, because otherwise the user will just see an ambiguous fileio/json error with no additional context and have to figure out that it is caused by the ICU dictionary